src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimContinuingOnlineApplicantForSpecNotificationHandler.java

```diff
@@ -67,9 +69,13 @@ private CallbackResponse notifyApplicantSolicitorForClaimContinuingOnline(Callba
 
         CaseData caseData = callbackParams.getCaseData();
 
+        String emailTemplateID = getMultiPartyScenario(caseData).equals(ONE_V_TWO_TWO_LEGAL_REP)
```

Comment:
Is this ticket only about 1 v 2 2 legal reps? 
If we have 1 v 2 1 legal rep then also I think the email should contain both defendants name.